pkg/config/system_probe.go

```diff
@@ -166,11 +166,13 @@ func InitSystemProbeConfig(cfg Config) {
 
 	// network_config namespace only
 	cfg.BindEnv(join(netNS, "enable_http_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP_MONITORING")
+	cfg.BindEnv(join(netNS, "enable_http2_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP2_MONITORING")
```

Comment:
It seems like this config is not needed, since the one in `service_monitoring_config` is the one that actually controls the feature.